src/patternfly/components/CustomSearchInput/custom-search-input.scss

```diff
@@ -0,0 +1,120 @@
+.pf-c-custom-search-input {
+  // Base border
+  --pf-c-custom-search-input__text--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-custom-search-input__text--before--BorderColor: var(--pf-global--BorderColor--300);
+
+  // Bottom border
+  --pf-c-custom-search-input__text--after--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-custom-search-input__text--after--BorderBottomColor: var(--pf-global--BorderColor--200);
+  --pf-c-custom-search-input--hover__text--after--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-custom-search-input__text--focus-within--after--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-custom-search-input__text--focus-within--after--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Search input
+  --pf-c-custom-search-input__text-input--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-custom-search-input__text-input--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-custom-search-input__text-input--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-custom-search-input__text-input--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-custom-search-input__text-input--MinWidth: 6ch;
+
+  // Icon
+  --pf-c-custom-search-input__icon--Left: var(--pf-global--spacer--sm);
+  --pf-c-custom-search-input__icon--Color: var(--pf-global--Color--200);
+  --pf-c-custom-search-input__text--hover__icon--Color: var(--pf-global--Color--100);
+  --pf-c-custom-search-input__text-input--focus--icon--Color: var(--pf-global--Color--100);
+  --pf-c-custom-search-input__icon--TranslateY: -50%;
+
+  // Utilities
+  --pf-c-custom-search-input__utilities--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-custom-search-input__utilities--child--MarginLeft: var(--pf-global--spacer--xs);
+  --pf-c-custom-search-input__utilities--c-button--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-custom-search-input__utilities--c-button--PaddingLeft: var(--pf-global--spacer--xs);
```

Comment:
could you add a negative margin to the left and right to make up for the lost area around the button ?

Reply:
not sure what you mean, can you mock it up and show me?